src/praisonai/tests/unit/test_approval_basic.py

```
@@ -219,16 +216,40 @@ def main():
     test_results = []
     
     # Run synchronous tests
-    test_results.append(("Imports", test_imports()))
-    test_results.append(("Configuration", test_approval_configuration()))
-    test_results.append(("Decorator", test_approval_decorator()))
-    test_results.append(("Tool Integration", test_tool_integration()))
-    test_results.append(("Agent Integration", test_agent_integration()))
+    try:
+        test_imports()
+        test_results.append(("Imports", True))
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)